packages/fx-core/src/common/azure-hosting/hostingFactory.ts

```diff
@@ -4,15 +4,15 @@
 import { AzureFunctionHosting } from "./azureFunctionHosting";
 import { AzureHosting } from "./azureHosting";
 import { BotServiceHosting } from "./botServiceHosting";
-import { HostType } from "./interfaces";
+import { ServiceType } from "./interfaces";
 
 const HostingMap: { [key: string]: () => AzureHosting } = {
-  [HostType.Function]: () => new AzureFunctionHosting(),
-  [HostType.BotService]: () => new BotServiceHosting(),
+  [ServiceType.Function]: () => new AzureFunctionHosting(),
+  [ServiceType.BotService]: () => new BotServiceHosting(),
```

Comment:
NIT: Although it's not fully related to the changes of the PR, how about using `{[key: string]: AzureHosting}` instead of using a function for the value? Would it be simpler?